test/servico/cst23Test.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package servico;
+
+import auxiliar.Perfil;
+import datamapper.PopulateDB;
+import java.text.ParseException;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import modelo.UsuarioModel;
+
+/**
+ *
+ * @author prgoes
+ */
+public class cst23Test {
+    
+    
+    @BeforeClass
+    public static void setUpClass(){ 
+        //
+    }   
+    
+    @AfterClass
+    public static void tearDownClass() {
+        //
+    }
+    
+    @Before
+    public void setUp() throws Exception {
+        PopulateDB.recreateDB("prosub", "root", "");
+        PopulateDB.populateUseCaseTest();
+        
+    }
+    
+    @After
+    public void tearDown() {
+        //
+    }
+
+    @Test
+    public void testeEditarUsuario() throws ParseException
+    {
+        LoginService loginService = new LoginService();
+        assertTrue(loginService.verificarUsuarioESenha(
+                "Administrador", "123456"));
+        AdministracaoDeUsuariosService administracaoDeUsuarioService =
+                new AdministracaoDeUsuariosService();
+        List<UsuarioModel> usuariosList = 
+                administracaoDeUsuarioService.listarUsuarios();
+        UsuarioModel usuario = usuariosList.get(3);         
+        assertEquals(usuario.Usuario, "Professor3");
+        assertEquals(usuario.Senha, "123456");
+        assertEquals(usuario.profile, Perfil.PROFESSOR);
+        try {
+            administracaoDeUsuarioService.editarUsuario(
+                    "SenhaEditadaTeste", Perfil.PROFESSOR, usuario.id);
+            UsuarioModel usuarioEditado = 
+                    administracaoDeUsuarioService.obterUsuario("Professor3");
+            assertEquals(usuarioEditado.Usuario, "Professor3");
+            assertEquals(usuarioEditado.Senha, "SenhaEditadaTeste");
+            
+            assertTrue(loginService.verificarUsuarioESenha(
+                    "Professor3", "SenhaEditadaTeste"));
+        } catch (Exception ex) {

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://www.codacy.com/app/calebepb/substituicao/file/2767610639/issues/source?bid=3308597&fileBranchId=3353664#l77)